### PR TITLE
test: Solidity fuzz repro

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/unittests/evm_other_test.cpp [hunk 1]
```diff
@@ -56,3 +56,12 @@ TEST_F(evm_other, loop_full_of_jumpdests)
...
...00000000000000000836003600481101515617d3457fe5b6020028101803560016080038336030381121515617d525760006000fd5b808301925050506003600481101515617d6757fe5b6020028101803560016020038336030381121515617d855760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617dab5760006000fd5b602081023603821315617dbe5760006000fd5b60028181101515617dcb57fe5b9050602002016020617de0919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617e125760549050618b07565b7f6c00000000000000000000000000000000000000000000000000000000000000836003600481101515617e4257fe5b6020028101803560016080038336030381121515617e605760006000fd5b808301925050506003600481101515617e7557fe5b6020028101803560016020038336030381121515617e935760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617eb95760006000fd5b602081023603821315617ecc5760006000fd5b60038181101515617ed957fe5b9050602002016020617eee919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617f205760559050618b07565b600482506004141515617f365760569050618b07565b6004826000600481101515617f4757fe5b6020028101803560016020038336030381121515617f655760006000fd5b8083019250508135905060208201915
```
Comment (bshastry) [inline test/unittests/evm_other_test.cpp]: Note to self: This is currently failing.

Comment (chfast) [reply test/unittests/evm_other_test.cpp]: The issue might be related to the code size. First of all this one is over 50k. evmone only supports code size up to 24k officially. I also tried to make it smaller, but the issue goes away if the code is trimmed by ~1k.

Comment (bshastry) [reply test/unittests/evm_other_test.cpp]: I agree. Am curious why this isn't an issue in evmone 0.1 (ref https://github.com/ethereum/solidity/issues/7776#issuecomment-558136555)

File: test/unittests/evm_other_test.cpp [hunk 2]
```diff
@@ -56,3 +56,12 @@ TEST_F(evm_other, loop_full_of_jumpdests)
     execute(code);
     EXPECT_GAS_USED(EVMC_SUCCESS, 7987882);
 }
+
+TEST_F(evm_other, solidity_deploy)
```
Comment (bshastry) [inline test/unittests/evm_other_test.cpp]: If you are looking for a more descriptive name for this unit test, you could perhaps use the name `abiv2_3d_bytes1_array_deploy`. The underlying contract is checks if abiv2 coding works as expected for a 3 dimensional `bytes1` array of type `bytes1[][4][4]`. The contract does not have an explicit constructor.

**Pull request closed without merging**